### PR TITLE
documentation added for specify setting by environment variable.

### PR DESCRIPTION
**Pull request by hramezani**
documentation added for specify setting by environment variable. #1385

### CODE REVIEW

**Comment by hramezani** [comment]
@berkerpeksag Documents added. i am ready for your comments :)

File: docs/gunicorn_ext.py [hunk 1]
```diff
@@ -18,6 +18,14 @@
...
 for reference on setting at the command line.
 
+.. note::
+
+    Settings can be specified by using environment variable
+    ``GUNICORN_CMD_ARGS``. All avaiable command line arguments can be used.
+    You may set multiple parameters by using space as separator. For example,
+    to specify the bind address and number or workers:
+
+    ``GUNICORN_CMD_ARGS ="--bind=127.0.0.1 --workers=3"``
```
Comment (berkerpeksag) [inline docs/gunicorn_ext.py]:
```rst
[...] or workers::

    $ GUNICORN_CMD_ARGS="--bind=127.0.0.1 --workers=3" gunicorn app:app
```

File: docs/gunicorn_ext.py [hunk 2]
```diff
@@ -18,6 +18,14 @@
 used in the configuration file. The command line arguments are listed as well
 for reference on setting at the command line.
 
+.. note::
+
+    Settings can be specified by using environment variable
+    ``GUNICORN_CMD_ARGS``. All avaiable command line arguments can be used.
+    You may set multiple parameters by using space as separator. For example,
```
Comment (berkerpeksag) [inline docs/gunicorn_ext.py]: "You may set multiple parameters by using space as separator." is not needed in my opinion.

File: docs/source/run.rst [hunk 3]
```diff
@@ -64,6 +64,8 @@ Commonly Used Arguments
   adjust the name of Gunicorn process as they appear in the process system
   table (which affects tools like ``ps`` and ``top``).
 
+Settings can be specified by using environment variable ``GUNICORN_CMD_ARGS``.
```
Comment (berkerpeksag) [inline docs/source/run.rst]: :ref:\`GUNICORN_CMD_ARGS \<settings\>\`

**Comment by hramezani** [comment]
@berkerpeksag Thanks, i fixed the problems.

File: docs/gunicorn_ext.py [hunk 4]
```diff
@@ -18,6 +18,13 @@
 used in the configuration file. The command line arguments are listed as well
 for reference on setting at the command line.
 
+.. note::
+
+    Settings can be specified by using environment variable
+    ``GUNICORN_CMD_ARGS``. All avaiable command line arguments can be used.
+    For example, to specify the bind address and number of workers::
+
+    $ GUNICORN_CMD_ARGS="--bind=127.0.0.1 --workers=3" gunicorn app:app
```
Comment (berkerpeksag) [inline docs/gunicorn_ext.py]: Just a style nit: This like needs to be indented as

```rst
For example, to specify the bind address and number of workers::

    $ GUNICORN_CMD_ARGS="--bind=127.0.0.1 --workers=3" gunicorn app:app
```

**Comment by hramezani** [comment]
@berkerpeksag Thanks, done.

**Comment by berkerpeksag** [comment]
I just added a versionadded annonation and merged, thanks!

**Pull request merged**